mdl/build/env.list

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2018 herd-mdl contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+action=deploy  # build|deploy
+build_from=remote # remote|local
+remote_repo_path=https://github.com/FINRAOS/herd-mdl.git
+branch=master
+s3_bucket=s3-bucket-name # provide an S3 bucket name to which you have read/write permissions
+s3_bucket_prefix=prefix
+default_stack_name=my-mdl-stack
+proxy=http://proxy-host:proxy-port
+parameter_file_name=default-parameters.json
+custom_tags=[{'Key': 'Custom Tag Key','Value': 'Custom Tag Value'}]
```

Comment:
move to param file

Reply:
I just realized that this is different than CFT parameters and has to be a separate input. Merging.